docs/source/user-guide/concepts.rst

```diff
@@ -6,6 +6,68 @@ Concepts
    :local:
    :depth: 1
 
+.. image:: /img/installing-with-conda.png
+    :align: right
+
+.. _installing-with-conda:
+
+Installing with conda
+=====================
+
+Conda install
+-------------
+* When you `conda install` a package that exists in your channels and has no dependencies, conda:
+
+  * looks at your configured channels (in priority)
+
+  * reaches out to the repodata associated with your channels/platform
+
+  * parses repodata to search for the package
+
+  * once the package is found, conda pulls it down and installs
+  
+
+.. _specifying-channels:
+
+Specifying channels
+-------------------
+
+* From the command line use `--channel`
+
+.. code-block:: bash
+
+  $ conda install scipy --channel conda-forge
+
+* From the command line use `--override-channels` to only search the specified channel
+
+.. code-block:: bash
+
+  $ conda search scipy --channel file:/<path to>/local-channel --override-channels
+
+* In .condarc with the keys
+
+  * channels: list of channels for conda to search for packages
+
+  * default_channels: normally pointing to channels at repo.continuum.io, sets the
+    list of "default channels"
+
+  * allow_other_channels: a boolean value that determines if the user may install
+    packages outside of the channels list
+
+  * channel_alias: sets an alias for a channel. For example, if `channel_alias: https://my.repo`
```

Comment:
Honestly, I'd advise removing  all of these except for ``channels``.  The others are for advanced applications, and they really don't belong in a "concepts" section.  Perhaps referencing the more-advanced section of the docs from here is a better way than showing those options completely here.